packages/store-ui/src/molecules/Bullets/stories/Bullets.mdx

```diff
@@ -0,0 +1,13 @@
+import { Preview, Props, Story, ArgsTable } from '@storybook/addon-docs/blocks'
```

Comment:
Almost there. Now, is it possible to limit the range to positive number only on both params?
![image](https://user-images.githubusercontent.com/1753396/123320191-d0d46580-d507-11eb-8f13-372e42d5ba52.png)

Reply:
Yeah! @victorhmp I think you can put like this `control: { type: 'range', min: 0 }`.

Reply:
Sure! Just did it 😄